pkg/serverless/logs/logs.go

```diff
@@ -85,10 +80,6 @@ const (
 	logTypePlatformReport = "platform.report"
 	// logTypePlatformLogsDropped is used when AWS has dropped logs because we were unable to consume them fast enough.
 	logTypePlatformLogsDropped = "platform.logsDropped"
-	// logTypePlatformLogsSubscription is used for the log messages about Logs API registration
-	logTypePlatformLogsSubscription = "platform.logsSubscription"
-	// logTypePlatformExtension is used for the log messages about Extension API registration
-	logTypePlatformExtension = "platform.extension"
```

Comment:
We were doing nothing with these two message types. I believe the code is cleaner and easier to read if these are removed entirely in favor of letting these messages fall into the `default` category during json unmarshalling.